series_solution.ipynb

```diff
@@ -0,0 +1,262 @@
+{
```

Comment:
<p>These are not the same data types as above</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ubsuny/CP1_PythonCpp/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>